packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/Update/Update.tsx

```diff
@@ -0,0 +1,199 @@
+
+    return theme[colorMap[$variant]];
+};
+
+type HighlightedProps = { $isHighlighted: boolean };
+
+const highlighted = css<HighlightedProps>`
+    ${({ $isHighlighted }) =>
+        $isHighlighted
+            ? ''
+            : css`
+                  opacity: 50%;
+              `}
+`;
+
+const Highlighted = styled.div<HighlightedProps>`
+    ${highlighted}
+`;
+
+type SuiteIconRectangle = {
+    $variant: UpdateVariant;
+    $isHighlighted: boolean;
+    $size: IconSize;
+};
+
+const SuiteIconRectangle = styled.div<SuiteIconRectangle>`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-left: 0.5px;
+    width: ${({ $size }) => getIconSize($size)}px;
+    height: ${({ $size }) => getIconSize($size)}px;
+
+    border-radius: ${borders.radii.xxs};
+    background-color: ${({ $variant, theme }) => mapVariantToIconColor({ $variant, theme })};
+
+    ${highlighted}
+`;
+
+const Relative = styled.div<{ $size: IconSize }>`
+    position: relative;
+    width: ${({ $size }) => getIconSize($size)}px;
+    height: ${({ $size }) => getIconSize($size)}px;
+`;
+
+type UsbCableFroTrezorIcon = {
+    $variant: UpdateVariant;
+    $size: IconSize;
+};
+
+const UsbCableFroTrezorIcon = styled.div<UsbCableFroTrezorIcon>`
+    border-left: 2px solid ${({ $variant, theme }) => mapVariantToIconColor({ $variant, theme })};
+    height: 5px;
+    position: absolute;
+    bottom: -4px;
+    left: ${({ $size }) => getIconSize($size) / 2}px;
+`;
+
+type DeviceUpdateIconProps = {
+    iconSize: IconSize;
+    updateStatus: UpdateStatus;
+    variant: UpdateVariant;
+};
+
+const DeviceUpdateIcon = ({ iconSize, updateStatus, variant }: DeviceUpdateIconProps) => {
+    const { device } = useDevice();
+
+    if (device?.features === undefined) {
+        return null;
+    }
+
+    return (
+        <Highlighted $isHighlighted={updateStatus !== 'up-to-date'}>
+            <Relative $size={iconSize}>
+                <Icon
+                    name={mapTrezorModelToIcon[device.features.internal_model]}
+                    size={iconSizes.medium}
+                    variant={variant}
+                />
+                <UsbCableFroTrezorIcon $variant={variant} $size={iconSize} />
+            </Relative>
+        </Highlighted>
+    );
+};
+
+type SuiteUpdateIconProps = {
+    iconSize: IconSize;
+    updateStatus: UpdateStatus;
+    variant: UpdateVariant;
+};
+
+const SuiteUpdateIcon = ({ iconSize, updateStatus, variant }: SuiteUpdateIconProps) => {
+    const theme = useTheme();
+
+    return (
+        <SuiteIconRectangle
+            $variant={variant}
+            $isHighlighted={updateStatus !== 'up-to-date'}
+            $size={iconSize}
+        >
+            <Highlighted $isHighlighted={updateStatus !== 'up-to-date'}>
+                <Icon name="trezor" size={iconSizes.small} color={theme['iconDefaultInverted']} />
+            </Highlighted>
+        </SuiteIconRectangle>
+    );
+};
+
+export const Update = () => {
```

Comment:
Could we rename it to something more descriptive or with some noun? I think it's easier when you are looking for component in your IDE.
Not sure but thinking about something like `ManageUpdate` or `UpdateStatus`